builtin/async/init.lua

```diff
@@ -1,4 +1,20 @@
 
+if core.get_cache_path == nil then -- TODO
+
+core.log("info", "Initializing Asynchronous environment for game")
+
+function core.job_processor(func, serialized_params)
+	local params = core.deserialize(serialized_params)
+
+	print('will call ' .. tostring(func) .. ' with ' .. tostring(#params) .. ' args')
+
+	local retval = {func(unpack(params))}
```

Comment:
This won't work properly with `return 1, nil, 2` (the table will contain just `1`).

Reply:
What will work then?